src/OmniSharp.Roslyn/OmniSharpWorkspace.cs

```diff
@@ -24,14 +24,27 @@ namespace OmniSharp
     [Export, Shared]
     public class OmniSharpWorkspace : Workspace
     {
-        public bool Initialized { get; set; }
+        public bool IsInitialized
+        {
+            get { return isInitialized; }
+            set
+            {
+                if (isInitialized == value) return;
+                isInitialized = value;
+                Initialized(isInitialized);
+            }
+        }
+
+        public event Action<bool> Initialized = delegate { };
```

Comment:
let's not make a breaking change here - the old property could stay named the same way and just adopt your new logic, while the event could be named differently

Reply:
Reverted names: Initialized and OnInitialized (new event)